app/scripts/modules/kubernetes/cluster/configure/CommandBuilder.js

```diff
@@ -80,6 +91,20 @@ module.exports = angular.module('spinnaker.kubernetes.clusterCommandBuilder.serv
         mode: mode,
       };
 
+      if (!command.capacity) {
+        command.capacity = {
+          min: command.targetSize,
+          max: command.targetSize,
+          desired: command.targetSize,
+        };
+      }
+
+      if (!command.scalingPolicy) {
```

Comment:
I see. You're also checking if the value is already defined (you would want to be careful about `cpuUtilization === 0`). You could potentially use

``` if (!_.has(command, 'scalingPolicy.cpuUtilization.target`) {
  _.set( ... )
}
```